### DISCUSSION TITLE
Few asks

### DISCUSSION THREAD

**Discussion by seba2282** [Open-Shell/Open-Shell-Menu#1924] [Q&A]
Will work still in next versions windows? After how Microsoft plan more that types programs to block?
When more languages? Classic Shell had multilanguages, but open shell so humble about it.

**Comment by ge0rdi** [comment]
> Will work still in next versions windows?

Hard to tell, we will see when that next version comes. `Open-Shell` seems to be working in Win11 Insider builds though.

> After how Microsoft plan more that types programs to block?

I'm not sure MS plans to _actively_ block this kind of customization software if it doesn't cause any troubles (there were some blocks because programs were crashing Explorer which is certainly not a thing anyone wants).

> When more languages? Classic Shell had multilanguages, but open shell so humble about it.

https://github.com/Open-Shell/Open-Shell-Menu/discussions/1733
https://github.com/Open-Shell/Open-Shell-Menu/discussions/1627
